src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -631,6 +631,16 @@ def load_arguments(self, _):
             ),
         )
         c.argument("nodepool_taints", validator=validate_nodepool_taints)
+        c.argument(
+            "nodepool_initialization_taints",
+            options_list=["--nodepool-initialization-taints", "--node-init-taints"],
+            is_preview=True,
+            validator=validate_nodepool_taints,
+            help=(
+                "comma-separated taints: <key1>=<value1>:<effect1>,<key2>=<value2>:<effect2>. "
+                "Pass "" to clear existing taints."
```

Comment:
I highly doubt you need to escape those two double quotes, could test this locally
```suggestion
                "Pass \"\" to clear existing taints."
```